Sources/SwiftParser/Statements.swift

```diff
@@ -162,6 +162,11 @@ extension Parser {
     var keepGoing: RawTokenSyntax? = nil
     var loopProgress = LoopProgressCondition()
     repeat {
+      if experimentalFeatures.contains(.trailingComma) {
+        if elements.count > 0, currentToken.tokenText == "else" {
```

Comment:
```suggestion
        if elements.count > 0, self.at(.else) {
```

Also, is the checking for `else` needed for `guard` statements? Would be good to add a comment for that, I think.

Reply:
That's right, I will add a comment. But maybe `self.at(.keyword(.else)), self.peek(isAt: .leftBrace)` is a better terminator for `guard`?

Reply:
I think just checking for `else` is sufficient. Just a comment would be good.